From 443638d3e53f1bd2bc9afbdf7164197961fe6e98 Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 2 Sep 2022 18:37:28 +0200
Subject: [PATCH] Comment shutdown call

---
 src/xoctave_interpreter.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/xoctave_interpreter.cpp b/src/xoctave_interpreter.cpp
index 81334cf..9a253aa 100644
--- a/src/xoctave_interpreter.cpp
+++ b/src/xoctave_interpreter.cpp
@@ -325,7 +325,8 @@ nl::json xoctave_interpreter::kernel_info_request_impl() {
 }
 
 void xoctave_interpreter::shutdown_request_impl() {
-	interpreter.shutdown();
+	// octave::interpreter::shutdown() became private with 7.0
+	// interpreter.shutdown();
 
 #ifndef NDEBUG
 	std::clog << "Bye!!" << std::endl;
@@ -390,4 +391,4 @@ json xoctave_interpreter::get_help_for_symbol(const std::string& symbol) {
 	return result;
 }
 
-}  // namespace xoctave
\ No newline at end of file
+}  // namespace xoctave
-- 
2.36.1

