From eb514fe1616fc356db29bbd7f4169eacf432a43a Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Fri, 2 Sep 2022 23:56:06 +0200
Subject: [PATCH] Do not enable fltk

---
 src/xoctave_interpreter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xoctave_interpreter.cpp b/src/xoctave_interpreter.cpp
index 9a253aa..ce03190 100644
--- a/src/xoctave_interpreter.cpp
+++ b/src/xoctave_interpreter.cpp
@@ -224,7 +224,7 @@ void xoctave_interpreter::configure_impl() {
 	interpreter.get_gtk_manager().register_toolkit("plotly");
 	interpreter.get_gtk_manager().load_toolkit(octave::graphics_toolkit(new xoctave::plotly_graphics_toolkit(interpreter)));
 
-	octave::feval("graphics_toolkit", ovl("fltk"));
+	// octave::feval("graphics_toolkit", ovl("fltk"));
 #ifdef NOTEBOOK_TOOLKIT_ENABLED
 	octave::feval("graphics_toolkit", ovl("notebook"));
 #else
-- 
2.36.1

